src/less/skeleton/skeleton.less

```diff
@@ -0,0 +1,204 @@
+@import "../variables/variables.less";
+@import "../mixins/private/skeleton-mixins.less";
+
+@skeleton-text-border-radius: 3px;
+@skeleton-text-height-small: 16px;
+@skeleton-text-height-large: 24px;
+
+.skeleton-avatar {
+    .skeleton-base(50%, var(--skeleton-height), var(--skeleton-height));
+}
+
+.skeleton-button {
+    .skeleton-base(20px, 40px);
+}
+
+.skeleton-button--small {
+    .skeleton-base(16px, 32px);
+}
+
+.skeleton-button--large {
+    .skeleton-base(24px, 48px);
+}
+
+.skeleton-text,
+.skeleton-text--small {
+    .skeleton-base(@skeleton-text-border-radius, @skeleton-text-height-small);
+}
+
+.skeleton-text::after,
+.skeleton-text--small::after {
+    height: @skeleton-text-height-small;
+    margin-top: calc(@skeleton-text-height-small + var(--spacing-100));
+}
+
+.skeleton-text--large {
+    .skeleton-base(@skeleton-text-border-radius, @skeleton-text-height-large);
+}
+
+.skeleton-text--large::after {
+    height: @skeleton-text-height-large;
+    margin-top: calc(@skeleton-text-height-large + var(--spacing-100));
+}
+
+.skeleton-text--multiline {
+    margin-bottom: var(--spacing-300);
+    position: relative;
+    width: calc(100% - var(--spacing-300));
+}
+
+.skeleton-text--multiline::after {
+    background: inherit;
+    content: "";
+    position: absolute;
+    width: calc(100% - var(--spacing-700));
+}
+
+.skeleton-text--large.skeleton--text--multiline {
```

Comment:
Had to use chaining here because spacing needs to be added when it meets both large and multiline conditions. May be we can use `skeleton-text--large-multiline` but preferred earlier approach.